zvariant/src/signature.rs

```diff
@@ -288,6 +293,21 @@ impl<'a> Signature<'a> {
 
         clone
     }
+
+    /// Returns the number of complete types for the signature.
```

Comment:
nit: `Returns` is redundant here and afaik inconsistent with general conventions. Hence why I removed it in my suggestion.

Reply:
ok